oshi-core/src/main/java/oshi/util/platform/linux/ProcPath.java

```diff
@@ -62,6 +62,10 @@
     public static final String UPTIME = PROC + "/uptime";
     public static final String VERSION = PROC + "/version";
     public static final String VMSTAT = PROC + "/vmstat";
+    public static final String THREADS_PATH = PROC + "/%d/task";
```

Comment:
I'm thinking `TASK` would be better than `THREAD` here.   Also the list does happen to be alphebetized, although that doesn't matter too much :)

Reply:
Sorry, i didn't check the alphabetical order of the list. Will fix this.

Reply:
It really doesn’t matter that much. :)